.gitignore

```diff
@@ -25,10 +25,6 @@ IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
 \#*
 .*.sw?
 
-# Monodevelop
-*.pidb
-*.userprefs
-
```

Comment:
Changes related to the Monodevelop IDE can stay.

Reply:
Monodevelop is EOL, might as well remove?